spec/std/path_spec.cr

```diff
@@ -164,6 +164,20 @@ describe Path do
     assert_paths("foo/./bar/.", [".", "foo", "foo/.", "foo/./bar"], &.parents)
     assert_paths("foo/bar/.", [".", "foo", "foo/bar"], &.parents)
     assert_paths("foo/bar/./.", [".", "foo", "foo/bar", "foo/bar/."], &.parents)
+    assert_paths("m/.gitignore", [".", "m"], &.parents)
+    assert_paths("m", ["."], &.parents)
+    assert_paths("m/", ["."], &.parents)
+    assert_paths("m//", ["."], &.parents)
+    assert_paths("m//a/b", [".", "m", "m//a"], &.parents)
```

Comment:
For posix paths this should be `[".", "m\\a"]`.

Reply:
I don't think so. Only if slashes in path were backslashes which aren't.